docs/development/FeatureFlag.md

```diff
@@ -0,0 +1,19 @@
+# Feature Flags - Policy and Intended Purposes for Sitrep Development
+
+## Why Feature Flags?
+We intend to implement and use Feature Flags to enable continuous development and testing of Sitrep in one branch.
+- Use case example: Development & testing of changes to existing features (e.g. a clock for the feed view of the Operation Journal).
+- Use case example: A specific feature is requested by an organization and they will beta-test it. Multiple features might be in development and tested by different users at the same time.
+- Use case example: A/B-Testing of UI-components or processes.
+
+
+## Sitrep Feature Flags Policy:
+1. Responsibility for each Feature Flag and adherence to this policy lies with the developer implementing it.
+2. Feature Flags shall only be implemented in front-end UI-elements, to enable or disable access to certain functionalities. 
+3. No Feature Flags must be implemented in back-end code. 
```

Comment:
If a Feature Flag in the UI generates different API calls on the backend, the backend must be able to deal with both calls independently. In some cases this means that a new API endpoint must be created.

Reply:
Suggestion:
`3. No Feature Flags must be implemented in back-end code. However, his does not apply to backend API endpoints needed for frontend Feature Flags.`